app/controllers/steps_controller.rb

```diff
@@ -0,0 +1,27 @@
+class StepsController < ApplicationController
+  include Wicked::Wizard::Translated
+
+  before_action :authenticate_user!
+
+  steps(*User.steps)
+
+  def show
+    render_wizard
+  end
+
+  def update
+    wizard_value = wizard_value(step)
+
+    if wizard_value != "stage_5"
+      if wizard_value >= current_user.highest_aasm_state
```

Comment:
Move to the next stage if the current page is greater than or equal to the further page we have visited.